.github/workflows/build-test.yml

```diff
@@ -20,12 +20,22 @@ jobs:
     runs-on:                    ${{ matrix.platform }}
     steps:
       - name:                   Checkout sources
-        uses:                   actions/checkout@v2
+        uses:                   actions/checkout@master
```

Comment:
any reason why v2 was the initial version? just curious

Reply:
There have been several versions before v2: https://github.com/actions/checkout/releases. Also submodule support has been added after v2: https://github.com/actions/checkout/commit/422dc4567157f4d62b665a8a288310365b1d194b.